src/utils/daos_hdlr.c

```diff
@@ -2397,7 +2397,7 @@ fs_copy(struct file_dfs *src_file_dfs,
 
 			D_ALLOC(buf, buf_size * sizeof(char));
```

Comment:
Unrelated but isn't this just buf_size?
```suggestion
			D_ALLOC(buf, buf_size);
```

Reply:
I think you're right.
@dsikich Could you verify this please?

Reply:
@daltonbohning well the sizeof a char should just be 1, so it is the same either way. I don't really see the issue being explicit here? But doesn't really matter to me either way.

Reply:
@dsikich oh at first I did not look at enough of the code. yeah, the sizeof(char) should be removed.

Reply:
@dsikich Is going to address this on https://github.com/daos-stack/daos/pull/4281